etc/chaos_supervisor.conf

```diff
@@ -1,5 +1,5 @@
 [program:chaos]
-command=/root/.virtualenvs/chaos/bin/python chaos.py
+command=bash chaos_wrapper.sh
```

Comment:
Yeah, but I don't know what the full path would be on production. Is there a particular reason why that would be better?